components/AddForm.js

```diff
@@ -1,9 +1,12 @@
 import styled from 'styled-components';
 import Buttons from './Buttons';
 import useStore from '../hooks/useStore';
+import { useState } from 'react';
 import { nanoid } from 'nanoid';
 
 export default function AddForm() {
+  const [inputValue1, setInputValue1] = useState('');
+  const [inputValue2, setInputValue2] = useState('');
```

Comment:
You could use another wording for these variables, for example `inputName` and `inputInformation` to make this more clear. But it's just my personal preference.